lms/djangoapps/grades/new/course_grade.py

```diff
@@ -3,10 +3,12 @@
 """
 
 from collections import defaultdict
+from logging import getLogger
+
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
 from lazy import lazy
```

Comment:
This says they should be separate groups: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order